package.json

```diff
@@ -110,7 +110,39 @@
             {
               "selector": "less",
               "colors": "css"
-            }
+            },
+            {
+              "selector": "html",
+              "colors": "css"
+            },
+		        {
+		        	"selector": "js",
+		        	"colors": [
+		        		"numhex",
```

Comment:
Let's keep `numhex` disabled by default. They are likely to match on too many existing hex literals which are not for colors